src/chat.rs

```diff
@@ -86,6 +86,14 @@ pub enum ProtectionStatus {
     ///
     /// All members of the chat must be verified.
     Protected = 1,
+
+    /// The chat was protected, but now a new message came in
+    /// which was not encrypted / signed correctly.
+    /// The user has to confirm that this is OK.
+    ///
+    /// We only do this in 1:1 chats; in group chats, the chat just
+    /// stays protected.
+    ProtectionBroken = 3,
```

Comment:
So that the numbers that are the same as in `VerifiedStatus` have the corresponding meaning:

Unverified ≙ 0 ≙ Unprotected
Verified ≙ 1 ≙ Protected _Protected should correspond to BidirectVerified, so it doesn't fit anyway_
BidirectVerified ≙ 2 ≙ _
_ ≙ 3 ≙ ProtectionBroken

However, it doesn't really fit anyway, so if I'd write the code again, I'd just use `2`.

But we already sent a preview APK to TeamUSEC, so changing the number now would need some database migration, so I'm in favor of just leaving it like this.